app/models/statcan_dataset.rb

```diff
@@ -0,0 +1,30 @@
+class StatcanDataset < ApplicationRecord
+  validates :statcan_url, presence: true, uniqueness: true, format: { with: URI::DEFAULT_PARSER.make_regexp }
```

Comment:
You can add your scope here
```suggestion
  scope :stale, ->(current_time = Time.current) {
  all.select { |dataset| dataset.needs_sync?(current_time) }
}
  validates :statcan_url, presence: true, uniqueness: true, format: { with: URI::DEFAULT_PARSER.make_regexp }
```